en/docs/quick-start-guides/configure-front-end-application-qsg.md

```diff
@@ -0,0 +1,44 @@
+
+In this step, you are adding the configurations needed for the web app to successfully invoke the **Reading List Service** REST API. These configurations need to be updated for each environment you deploy the web app. Here you will be updating the configurations for the development environment.
+
+!!! note
+        The web application is reading the environment-specific configurations from the `window` object at runtime. This is done via the `config.js` file in the root of the web application. In this section, we are mounting the `config.js` file for the development environment. You will need to do the same for other environments as well when you deploy your web application to multiple environments.
+
+To configure the front-end application, follow the steps given below.
+
+1. While on the web application component page. Click and expand **Dev Ops** in the left menu.
+2. Select **Configs and Secrets** sub menu.
```

Comment:
Shall we add this also as part of step one as follows?
While on the web application component page, click **DevOps** in the left navigation menu and then click **Configs and Secrets**.

Reply:
done